github/labels/default.json

```diff
@@ -47,6 +47,10 @@
     "name": "FUN",
     "color": "006b75"
   },
+  {
+    "name": "release",
+    "color": "006b75"
```

Comment:
It's the same color as `FUN` label, is it what you want ?

Reply:
Yes! Unless you have another suggestion?

Reply:
Yes something green, maybe like `196811` ?

Reply:
![](https://media.giphy.com/media/Hq2XDestpCXq8/giphy.gif)

Reply:
![](https://media.giphy.com/media/x7gjmBuaHrWak/giphy.gif)

Reply:
Done.